service/client_user_test.go

```diff
@@ -9,8 +9,10 @@ import (
 )
 
 func Test_userDial(t *testing.T) {
```

Comment:
Did you use underscore for test names on purpose? Because everywhere else is pascal case.

Reply:
There was some naming collision. I changed it back to camel-case so the function above is now `TestClientUserDial`